lms/static/js/verify_student/views/webcam_photo_view.js

```diff
@@ -314,6 +317,18 @@
                 .attr('aria-disabled', !isEnabled);
         },
 
+        setSubmitButtonFocused: function( ){
+            $( this.submitButton )
+                .trigger('focus');
+            $('#submit').attr('title', gettext( 'Photo Captured successfully '));
+        },
+
+        capture_sound: function(){
+            var audio = document.createElement('audio');
+            audio.src = '/static/audio/camera_capture.wav';
```

Comment:
Unfortunately, this URL isn't going to work correctly in production with our CDN.  You'll need to pass the URL to the JavaScript from the server, most likely using `static.url()` in the template to retrieve the full URL.  This URL is preferable because (a) it will come from the CDN, which is likely to be faster than loading it directly from our application servers, and (b) it will contain a cache-busting hash in the filename, so if we change the sound file clients will automatically use the new version.